ToDoApp/ToDoApp.xcodeproj/project.pbxproj

```diff
@@ -7,7 +7,10 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		3A4C5F152938393A002EEE92 /* ToDoService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A4C5F142938393A002EEE92 /* ToDoService.swift */; };
+		3A4C5F5329398054002EEE92 /* ToDoService.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A4C5F142938393A002EEE92 /* ToDoService.swift */; };
+		3A4C5F8D2939BA2D002EEE92 /* CreateTaskView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A4C5F8C2939BA2D002EEE92 /* CreateTaskView.swift */; };
+		3A4C5F8F2939BC33002EEE92 /* CreateTaskView-ViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3A4C5F8E2939BC33002EEE92 /* CreateTaskView-ViewModel.swift */; };
```

Comment:
it is not common to have `-` character in file names. it would be better to name it either CreateTaskViewViewModel or if this is an extension to the CreateTaskViewType then there is a common pattern to name extensions with `+` character. like `CreateTaskView+ViewModel`

Reply:
This has been fixed.

Reply:
should be `CreateTaskView+ViewModel`, not `CreateTaskView+ModelView`